cli/parser/parser.go

```diff
@@ -98,19 +98,21 @@ var preBazel7ExpansionOptions = map[string]struct{}{
 	"persistent_multiplex_android_resource_processor": struct{}{},
 	"persistent_android_dex_desugar":                  struct{}{},
 	"persistent_multiplex_android_dex_desugar":        struct{}{},
-	"start_app":                     struct{}{},
-	"debug_app":                     struct{}{},
-	"java_debug":                    struct{}{},
-	"remote_download_minimal":       struct{}{},
-	"remote_download_toplevel":      struct{}{},
-	"long":                          struct{}{},
-	"short":                         struct{}{},
-	"expunge_async":                 struct{}{},
-	"experimental_spawn_scheduler":  struct{}{},
-	"experimental_persistent_javac": struct{}{},
-	"null":                          struct{}{},
```

Comment:
That's an interesting option, I didn't know this existed.

Reply:
yeah, lol, that one also surprised me; it sounds made-up